tests/integration/compose/docker_compose_iceberg_hms_catalog.yml

```diff
@@ -52,16 +52,17 @@ services:
   mc:
     depends_on:
       - minio
-    image: minio/mc
+    # Stick to version with "mc config"
+    image: minio/mc:RELEASE.2025-04-16T18-13-26Z
```

Comment:
I've also pinned all other docker images that was using in this tests

And about this place, it fails for me, since new `mcli` does not `config` command (`alias` should be used instead, but we stick with older version for now)